Artiom/src/main/java/Expeliarmus.java

```diff
@@ -0,0 +1,34 @@
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+
+public class Expeliarmus {
```

Comment:
Is it "Primitives" task?
Could you please remove not Primitive tasks from this PR

Reply:
done